core/pom.xml

```diff
@@ -53,7 +53,22 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
We should not add these on a library. We should remove logback-core also from this pom.xml

Reply:
ok, removed logback completely next.